openaq_fastapi/openaq_fastapi/routers/countries.py

```diff
@@ -1,45 +1,84 @@
 import logging
-
+from typing import Union, List
 from fastapi import APIRouter, Depends, Query
 from enum import Enum
 from ..db import DB
 from ..models.queries import APIBase, Country
 from openaq_fastapi.models.responses import (
     CountriesResponse,
-    converter
+    CountriesResponseV1,
 )
-import jq
+
 logger = logging.getLogger("countries")
 
 router = APIRouter()
 
 
 class CountriesOrder(str, Enum):
-    country = "country"
+    code = "code"
+    name = "name"
+    locations = "locations"
     firstUpdated = "firstUpdated"
     lastUpdated = "lastUpdated"
-    locations = "locations"
+    parameters = "parameters"
+    count = "count"
+    cities = "cities"
+    sources = "sources"
+
+
+class CountriesOrderV1(str, Enum):
+    code = "code"
     count = "count"
+    locations = "locations"
+    cities = "cities"
+    name = "name"
+
+
+class CountriesV1(APIBase):
+    order_by: CountriesOrderV1 = Query(
+        "code", description="Order by a field e.g. ?order_by=code", example="code"
+    )
+    limit: int = Query(
+        200,
+        description="Limit the number of results returned. e.g. limit=200 will return up to 200 results",
+        example="200",
+    )
+
+    def where(self):
+        wheres = []
+        for f, v in self:
+            if v is not None:
+                if f == "country":
+                    wheres.append(
+                        """
+                        c.iso = ANY(:country)
+                        """
+                    )
+        if len(wheres) > 0:
+            return (" AND ").join(wheres)
+        return " TRUE "
 
 
 class Countries(Country, APIBase):
     order_by: CountriesOrder = Query(
-        "country",
-        description="Order by a field e.g. ?order_by=country",
-        example="country"
+        "name", description="Order by a field e.g. ?order_by=name", example="name"
     )
     limit: int = Query(
         200,
```

Comment:
Change this to 100 so its consistent with other `limit` defaults also change to 100 in the example and description strings below

Reply:
👌🏼